e2e-tests/playwright/support/ui/components/channels/message_priority.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, Locator} from '@playwright/test';
+
+export default class MessagePriority {
```

Comment:
Good thing that this component is added. It's good to be part of the `ChannelsPage`. However, I don't see it used elsewhere.